packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -63,8 +63,10 @@ html.interface-interface-skeleton__html-container {
 	overscroll-behavior-y: none;
 
 	// Footer overlap prevention
-	@include break-medium() {
-		padding-bottom: $button-size-small + $border-width;
+	.show-block-breadcrumbs & {
```

Comment:
Similar to above question regarding className + conditional element appending relying on the same logic being possibly redundant?

Reply:
It prevents inserter overscroll. Inserter (and I think the navigation panel too) overlp the footer. That's why we need this rule here.